src/test/skript/tests/syntaxes/effects/EffEquip.sk

```diff
@@ -1,12 +1,65 @@
 test "equip effect":
-	spawn a zombie at spawn of "world":
-		set {_entity} to event-entity
 
-	equip {_entity} with a diamond chestplate
-	assert chestplate of {_entity} is a diamond chestplate with "entity was not wearing a diamond chestplate"
-	set chestplate of {_entity} to air
+	# === ARMOUR ===
 
-	equip {_entity} with a diamond chestplate named "Test"
-	assert chestplate of {_entity} is a diamond chestplate named "Test" with "entity was not wearing a named diamond chestplate"
+	spawn a zombie at (spawn of world "world"):
+		set {_z} to entity
```

Comment:
not insanely important but any chance we can get a better variable names
`{_z}` could be a zombie, z-coord, zebra, a letter you never really know